test/integration/targets/ansible-vault/runme.sh

```diff
@@ -552,21 +552,16 @@ sudo chmod 000 "${MYTMPDIR}/unreadable"
 ansible-vault encrypt_string content
 ansible-vault encrypt_string content --encrypt-vault-id id3
 
-set +e
-
 # Try to use a missing vault password file
 ansible-vault encrypt_string content --encrypt-vault-id id1 2>&1 | tee out.txt
```

Comment:
If you want to verify the command returns a non-zero exit code, you can use something like this:

```suggestion
if ansible-vault encrypt_string content --encrypt-vault-id id1 > out.txt 2>&1; then
  echo "command did not fail"
  exit 1
fi

cat out.txt
```

Reply:
Updated as suggested